docs/en/interfaces/formats.md

```diff
@@ -99,10 +99,34 @@ The minimum set of characters that you need to escape when passing data in TabSe
 
 Only a small set of symbols are escaped. You can easily stumble onto a string value that your terminal will ruin in output.
 
-Arrays are written as a list of comma-separated values in square brackets. Number items in the array are fomratted as normally, but dates, dates with times, and strings are written in single quotes with the same escaping rules as above.
+Arrays are written as a list of comma-separated values in square brackets. Number items in the array are formatted as normally, but dates, dates with times, and strings are written in single quotes with the same escaping rules as above.
 
 [NULL](../query_language/syntax.md) is formatted as `\N`.
 
+Elements of [Nested](../data_types/nested_data_structures/nested.md) structures are formatted as arrays each.
```

Comment:
Is is not only a format issue. Array of structures is converted to arrays of nested fields